servers/rendering/renderer_viewport.h

```diff
@@ -117,6 +117,8 @@ class RendererViewport {
 
 		bool transparent_bg = false;
 
+		uint32_t canvas_cull_mask;
```

Comment:
needs initialization to default value here

Reply:
To clarify, even if in other members they are not initialized due to legacy (pre C++11) code, for all new code we ask to add default initializers.